apps/prairielearn/elements/pl-integer-input/pl-integer-input.mustache

```diff
@@ -15,9 +15,10 @@
             size="{{size}}"
             {{^editable}}disabled{{/editable}}
             {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}}
-            aria-describedby="pl-integer-input-{{uuid}}-suffix"
             placeholder="{{placeholder}}"
-            aria-labelledby="pl-integer-input-{{uuid}}-label"
+            {{#suffix}}aria-describedby="pl-integer-input-{{uuid}}-suffix"{{/suffix}}
+            {{#label}}aria-labelledby="pl-integer-input-{{uuid}}-label"{{/label}}
+            {{^label}}aria-label="Variable {{name}}"{{/label}}
```

Comment:
We shouldn't assume that this is a variable, or that `name` has any human-readable meaning. Let's just avoid rendering anything if a label isn't specified. We should add a note to the element docs that `label` is strongly encouraged for proper accessibility.

Reply:
785357da395b470176b257d2ad9cac4482f141ef